content/logs/processing/attributes_naming_convention.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Attributes Naming Convention
+kind: documentation
+description: "Parse your logs using the Grok Processor"
+aliases:
+    - logs/parsing/
+further_reading:
+- link: "logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "logs/faq/log-parsing-best-practice"
+  tag: "FAQ"
+  text: Log Parsing - Best Practice
+- link: "logs/logging_without_limits"
+  tag: "Documentation"
+  text: Control the volume of logs indexed by Datadog
+---
+
+## Tl;Dr
+
+One may ask, why do we need a naming convention for Log attributes?
+
+* Various technologies tend to use different attribute names for the same meaning.
+* This usually generates too many attributes which can lead to: user confusion and un-ability to correlate logs across sources.
+* Datadog Integrations rely on the following attribute naming convention.
+* We advise you to use the following reference when you parse your custom formats & sources in order to leverage fully all functionalities of your Datadog application.
+
+## Overview
+
+Centralizing logs from various technologies and applications tends to generate tens or hundreds of different attributes in a Log Management environment. Especially when many users - each one with its own personal usage- are working with it. 
+This generates confusion among users. For instance, a client IP can have the following attributes within your logs: `clientIP`, `client_ip_address`, `remote_address`, `client.ip`, and many other combinations are possible...
+In this context, knowing which attribute corresponds to the logs you are trying to filter on or correlating this same proxy logs to the web application logs just behind it is a cumbersome. 
```

Comment:
In this context, it can be cumbersome to know which attributes correspond to the logs you are trying to filter on, or correlate proxy logs to web application logs.